react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -84,7 +74,9 @@ dependencies {
     } else {
         implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-svg', readHashedVersion('../../package.json', 'react-native-svg', 'dependencies')))
         implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-aztec', submoduleGitHash('../../', 'react-native-aztec')))
-        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-recyclerview-list', 'a2e8ca550412504c32218fd473c55f696f18f1f7'))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-recyclerview-list', '7286f7494a4fc8bdae987ecbf56265c508caf3c9'))
     }
-    implementation 'com.facebook.react:react-native:+'
+
+    def rnVersion = readReactNativeVersion('../package.json', 'peerDependencies')
+    implementation "com.facebook.react:react-native:${rnVersion}"
```

Comment:
[Following the discussion](https://github.com/wordpress-mobile/react-native-aztec/pull/100#discussion_r242570153) in the react-native-aztec PR, let's leave the version used for RN to `+` when using the source level mode. I guess in this case we can just use the `if (rootProject.ext.buildGutenbergFromSource)` already in place a few lines above to host it.